### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by ASM995**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/---report-a-bug.md [hunk 1]
```diff
@@ -26,13 +26,15 @@ If you can, add screenshots to show the problem. If you need to show a code exam
 ## 🖥 Desktop
  - OS: [e.g. macOS]
  - Browser: [e.g. chrome, safari]
- - Version: [e.g. 22]
+ - Browser Version: [e.g. 22]
+- ICDS Website Version [V1, V2]
```
Comment (ad9242) [inline .github/ISSUE_TEMPLATE/---report-a-bug.md:30]:
```suggestion
- ICDS Website Version [e.g. V1, V2]
```
added "e.g." to be consistent with other details. 

Just a question though, does "V1" mean the "old" v1 site? Just wondering if we say something like "As displayed on the site top navigation" ?

File: .github/ISSUE_TEMPLATE/------suggest-a-development-implementation-improvement-.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 name: "\U0001F9D1‍\U0001F4BB Suggest a Development/implementation improvement "
 about: Suggest something for the UI Kit or website code.
 title: ''
-labels: ''
+labels: bug
```
Comment (ad9242) [inline .github/ISSUE_TEMPLATE/------suggest-a-development-implementation-improvement-.md:5]: is "bug" the right label for this type? should it be "enhancement"?

Comment (ASM995) [reply .github/ISSUE_TEMPLATE/------suggest-a-development-implementation-improvement-.md:5]: This was meant to be on the bug ticket. I will add enhancement to this template and bug to our bug template.

File: .github/ISSUE_TEMPLATE/---report-a-bug.md (hunk 1 above)
Comment (ASM995) [reply .github/ISSUE_TEMPLATE/---report-a-bug.md:30]: Yes will add that in too.

**Comment by ASM995** [comment]
Updates made in a new PR - #105

**Pull request closed without merging**